template-for-recording/record_screen_and_upload.bat

```diff
@@ -32,9 +36,15 @@ echo location of your Java installation.
 goto fail
 
 :findJavaFromJavaHome
-set JAVA_HOME=%JAVA_HOME:"=%
+set "JAVA_HOME=%JAVA_HOME:"=%
 set JAVA_EXE=%JAVA_HOME%/bin/java.exe
```

Comment:
This is where the odd slashes are coming from. I think the path should add backslashes.

Reply:
Changed now - maybe the first time it's going thru this path since the script was written